docs/docs/api-files.md

```diff
@@ -1,10 +1,10 @@
 ---
-title: API Files
+title: Pliki API
 ---
 
-Gatsby uses 4 files in the root of your project to configure your site and control its behavior. All of these files are optional.
+Gatsby używa 4 plików w głownym katalogu projektu do konfiguracji twojej strony oraz kontroli tego jak się zachowuje. Wszytskie z tych plików są opcjonalne.
```

Comment:
Wszytskie z tych plików są opcjonalne.
change to:
Wszystkie z tych plików są opcjonalne.
// small typo